libs/isograph-react/src/core/makeNetworkRequest.ts

```diff
@@ -149,3 +164,70 @@ type NetworkRequestStatus =
       readonly kind: 'UndisposedComplete';
       readonly retainedQuery: RetainedQuery;
     };
+
+function makeNetworkRequestIfNecessary(
+  environment: IsographEnvironment,
+  artifact: RefetchQueryNormalizationArtifact | IsographEntrypoint<any, any>,
+  variables: Variables,
+  normalizationAstLoader: () => Promise<NormalizationAst>,
+) {
+  return flatMapNetworkRequest(
+    [wrapPromise(normalizationAstLoader()), () => {}],
```

Comment:
Do we need to store this like `EntrypointLoader`? https://github.com/isographlabs/isograph/blob/c83d49aa02c4bd0a07853848038cd528519cbd30/libs/isograph-react/src/core/IsographEnvironment.ts#L180?

Why do we even store it? Calling `import` multiple times doesn't trigger extra requests
```tsx
await import("./foo.js") === await import("./foo.js") // true
```